### PR TITLE
Add Mridul as Celeborn committer

### PR DESCRIPTION
**Pull request by mridulm**

### CODE REVIEW

**Review by AngersZhuuuu — APPROVED** [review]
Congratulations !

**Comment by waitinfuture** [comment]
Congratulations!

File: docs/community/project_management_committee.md [hunk 1]
```diff
@@ -57,12 +57,14 @@ A contributor who contributes enough code to Celeborn will be promoted to a comm
...
-| Binjie Yang     | zwangsheng    | binjieyang@apache.org     |
-| Shaoyun Chen    | cxzl25        | csy@apache.org            |
-| Xiaofeng Jiang  | SteNicholas   | nicholasjiang@apache.org  |
-| Yifan Xia       | nafiyAix      | nafiyaix@apache.org       |
-| Yihe Li         | onebox-li     | liyihe@apache.org         |
-| Zhongqiang Chen | zhongqiangczq | zhongqiangchen@apache.org |
+| Name                | GitHub ID     | Apache ID                 |
+|:--------------------|:--------------|:--------------------------|
+| Binjie Yang         | zwangsheng    | binjieyang@apache.org     |
+| Chandni Singh       | otterc        |                           |
```
Comment (mridulm) [inline docs/community/project_management_committee.md]: +CC @otterc, added you here as well.
Not sure about your apache id though !

**Review by SteNicholas — APPROVED** [review]
Congratulations!

**Comment by onebox-li** [comment]
Congratulations!

**Comment by otterc** [comment]
Congratulations @mridulm

File: docs/community/project_management_committee.md (hunk 1 above)
Comment (otterc) [reply docs/community/project_management_committee.md]: Thanks @mridulm. My apache id is csingh@apache.org

**Comment by mridulm** [comment]
Thanks everyone !
Do we merge this via github's `Squash and merge` ? I did not see a `merge_pr.py`

**Comment by AngersZhuuuu** [comment]
Yea, just squash and merge

**Pull request merged**